### PR TITLE
fix: re-enable execution tests

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

They don't do a lot but may as well re-enable...

### CODE REVIEW

File: bitcoin/src/lib.rs [hunk 1]
```diff
@@ -145,10 +145,11 @@ pub trait BitcoinCoreApi {
 
 pub struct LockedTransaction {
     pub transaction: Transaction,
-    _lock: OwnedMutexGuard<()>,
+    _lock: Option<OwnedMutexGuard<()>>,
```
Comment (sander2) [inline bitcoin/src/lib.rs:148]: Why this change? I don't see None being used anywhere?

Comment (gregdhill) [reply bitcoin/src/lib.rs:148]: https://github.com/interlay/polkabtc-clients/pull/55/files#diff-55b26c6a59093ae5631a640da179b541dc72b0da88ad229fcb19a1dfecacf1ddR620

**Review by sander2 — APPROVED** [review]
